### PR TITLE
Fix OpenGraph Metadata and RSS Feed Item URL

### PR DESCRIPTION
**Pull request by justinyoo**
This PR includes some fixes including:

* There are duplicated messages in `About.vue`
  * Refactored duplicated messages into computed properties
* `siteURL + slug` doesn't represent the complete URL
  * Changed `slug` to `path` to show the correct URL on:
    * OpenGraph metadata for the post URL in `Post.vue`
    * RSS feed item URL in `gridsome.config.js`

It would be great if you take a look and merge it.

### CODE REVIEW

File: gridsome.config.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ module.exports = {
         feedItemOptions: node => ({
           title: node.title,
           description: node.description,
-          url: 'https://gridsome-starter-bleda.netlify.com/' + node.slug,
+          url: 'https://gridsome-starter-bleda.netlify.com/' + node.path,
```
Comment (cossssmin) [inline gridsome.config.js]: Remove trailing slash on URL, right now we're getting item URLs with a double slash, like this:

`https://gridsome-starter-bleda.netlify.com//writing-posts/`

This should work:

```
url: 'https://gridsome-starter-bleda.netlify.com' + node.path,
```

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,4 @@ node_modules
 dist
 static/feed.xml
```
Comment (cossssmin) [inline .gitignore]: Remove empty line

**Comment by cossssmin** [comment]
Hey, thanks for the PR! Just a couple things and I think we can merge 👍

File: gridsome.config.js (hunk 1 above)
Comment (justinyoo) [reply gridsome.config.js]: Removed the trailing slash.

File: .gitignore (hunk 2 above)
Comment (justinyoo) [reply .gitignore]: Removed the empty line.

**Comment by justinyoo** [comment]
@cossssmin Thanks for the review! I've updated both files as requested.

**Comment by cossssmin** [comment]
Looking good, thank you!

**Pull request merged**